scripts/tools/combine-tools.js

```diff
@@ -41,15 +45,15 @@ const getFinalTool = async (toolObject) => {
 
 }
 
-const combineTools = async (automatedTools) => {
+const combineTools = async (automatedTools, manualTools) => {
     for (const key in automatedTools) {
         let finalToolsList = [];
         if (automatedTools[key].toolsList.length) {
             for (const tool of automatedTools[key].toolsList) {
                 finalToolsList.push(await getFinalTool(tool))
             }
         }
-        if (manualTools[key].toolsList.length) {
+        if (manualTools[key] && manualTools[key].toolsList.length) {
```

Comment:
What if `automatedTools` doesn't have key that is in `manualTools`? 😄 Because in current code we can "skip" some tools defined in the `manualTools`.

Reply:
We already have defined certain categories in the category list so if anyone adds a new category in manual-tools, it has to be updated on the category-list file first, because it is connected to frontend implementation also.

Reply:
if something that is not in category list is added to manual list, it is just ignored, which is good

if something is added to manual list, and category list is updated, then it automatically is in automated list


or

tl;dr - there will be no issues. The only reason I extended this if is in case there is something in category list, but was not added to manual list

Reply:
Couldn't this logic be made more complicated? 😆 But ok, you guys are more familiar with the implementations, so it's ok for me.

Reply:
Yeah, I agree it is quite complicated for me also. If don't revise it for months, I'll forget the logic behind this implementation 😅

Reply:
It can be fixed in other PR, I don't want to block you guys :)